README.md

```diff
@@ -59,15 +59,17 @@ After that, you can run ESLint on any file or directory like this:
 
 ## Configuration
 
-After running `npm init @eslint/config`, you'll have an `.eslintrc` file in your directory. In it, you'll see some rules configured like this:
-
-```json
-{
-    "rules": {
-        "semi": ["error", "always"],
-        "quotes": ["error", "double"]
+After running `npm init @eslint/config`, you'll have an `eslint.config.js` or `eslint.config.mjs` file in your directory. In it, you'll see some rules configured like this:
+
+```js
+module.exports = [
+    {
+        "rules": {
+            "semi": ["error", "always"],
+            "quotes": ["error", "double"]
```

Comment:
wondering if we can replace these rules with some others. (formatting rules have been deprecated).

Reply:
`@eslint/config` will no longer have an option to configure `semi`, `quotes`, and other formatting rules?

Perhaps a config like this would be a more representative example:

```js
import pluginJs from "@eslint/js";

export default [
  pluginJs.configs.recommended,
];
```